_config.yml

```diff
@@ -3,27 +3,45 @@ title: ActiveMQ
 email:
 author:
 description:
-        url: http://apache.activemq.org
+  url: http://apache.activemq.org
 
 # Build settings
 permalink: none
 # permalink: :title/
 sass:
   sass_dir: css/
 
+collections:
+  releases:
+    output: true
+    permalink: /:name
+
+current_releases:
+  - 5.16
+  - 5.15
+
```

Comment:
Another thought...since there are multiple components and others could similarly use this approach, we should probably make these variable namings more specific for clarity...e.g. 5x_releases, current_5x_releases